iris_grib/_save_rules.py

```diff
@@ -1489,16 +1493,16 @@ def data_section(cube, grib):
 
     if fill_value is None:
         # Disable missing values in the grib message.
-        gribapi.grib_set(grib, "bitmapPresent", 0)
+        eccodes.codes_set(grib, "bitmapPresent", 0)
     else:
         # Enable missing values in the grib message.
-        gribapi.grib_set(grib, "bitmapPresent", 1)
-        gribapi.grib_set_double(grib, "missingValue", fill_value)
+        eccodes.codes_set(grib, "bitmapPresent", 1)
+        eccodes.codes_set_double(grib, "missingValue", fill_value)
 
-    gribapi.grib_set_double_array(grib, "values", data.flatten())
+    eccodes.codes_set_double_array(grib, "values", data.flatten())
 
     # todo: check packing accuracy?
-#    print("packingError", gribapi.getb_get_double(grib, "packingError"))
+#    print("packingError", eccodes.getb_get_double(grib, "packingError"))
```

Comment:
It looks like this was just a typo when it was first added. `getb_get_double` doesn't exist so this should be :

```suggestion
#    print("packingError", eccodes.get_get_double(grib, "packingError"))
```